src/Nncase.Core/PatternMatch/PatternUtility.cs

```diff
@@ -98,20 +101,53 @@ public static CallPattern IsWildcardCall<T>(string callName, string opName, Patt
 
     public static VArgsPattern ParamsWithArg(Pattern argPattern) => new VArgsPattern(
```

Comment:
Add document. This method actual want to match args have one or more match with `argPattern`, maybe can change name to `ContainsArg`.

Reply:
This part of the code is just formatted and will add document  in the new pr